x/batch/msg.go

```diff
@@ -0,0 +1,25 @@
+package batch
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/x"
+)
+
+const (
+	PathExecuteBatchMsg = "batch/execute"
+)
+
+var _ weave.Msg = (*ExecuteBatchMsg)(nil)
+
+func (*ExecuteBatchMsg) Path() string {
+	return PathExecuteBatchMsg
+}
+
+func (m *ExecuteBatchMsg) Validate() error {
+	for _, v := range m.Messages {
```

Comment:
req: Please limit the number of messages allowed in a batch. Also batches must not include other batches but that is probably defined in protobuf.

Reply:
Maybe 10? But yeah, any constant is good to enforce and easily adjust later

Reply:
@ethanfrey 10 it is, going to be a constant so we can change it.
I keep wondering if it might be cool to include this config variable in genesis and default to something when it does not exist?

Reply:
> I keep wondering if it might be cool to include this config variable in genesis and default to something when it does not exist?

Yes, nice idea. We can do this a few other places, to read constants from the genesis file. Issue is, we only get the genesis file on `init`, so for restart, this needs to be persisted to the kvstore.

Please make some reusable code for this, and happy to use this other places. I started doing this ages ago for fees, but never finished the PR as I had 1000 other things and it wasn't priority. Maybe this code gives some ideas, but it is not polished or tested fully, so don't copy, just compare those ideas to yours.
https://github.com/iov-one/weave/pull/41

Reply:
@ethanfrey I've been wondering: why don't we use initializer for that as well?
And then query the kv store for values we need.

This has it's drawbacks though, but if we need to store this in KV, why bother?

Reply:
I need to think about this a bit more though